### PR TITLE
Strip output redirection

### PR DESCRIPTION
**Pull request by EvyBongers**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const status_failed = '✕';
 
 function shell(command, options) {
   core.debug(`Running command: ${command}`);
-  const sh = spawnSync('/bin/sh', ['-c', `${command} 2>&1`], {
+  const sh = spawnSync('/bin/sh', ['-c', command], {
```
Comment (tun0) [inline index.js:11]: Why this change? The redirection was intentional as you want to capture both `stdout` and `stderr` of `command`. With the previous setup the output (and errors) of `command` would end up in `stdout` and output (read: errors) of `/bin/sh` would end up in `stderr`.

Comment (EvyBongers) [reply index.js:11]: Two-fold:
1. You already capture stderr as well: https://github.com/Pararius/action-terraform/pull/9/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R18-R22
2. The string specified there is part of the command arg passed to `/bin/sh`, whereas it should be `/bin/sh -c '<command>' 2&>1 if there really is a need to redirect `stderr` to `stdout`.

Comment (tun0) [reply index.js:11]: There's 2 different `stderr`s at play here: the `stdout` of `sh` and the `stdout` of `command`. As stated in my previous comment: I don't want to redirect the `stderr` of `sh`, but only of `command`.

Comment (EvyBongers) [reply index.js:11]: `sh` does not eat the stderr of commands it executes and pass it on on stderr? 🤯

Comment (tun0) [reply index.js:11]: It does, by default, but that's not what I want. The goal was to keep errors of `command` and `sh` separate (by redirecting those of `command` to `stdout`) :wink: By default they both end up in (the same) `stderr`.

Comment (EvyBongers) [reply index.js:11]: You _want_ `stderr` output that `terraform` (or any command) might return on `stdout`? Why?

Comment (tun0) [reply index.js:11]: One of the main reasons is to keep output chronological as much as possible: 
```
stdout:
[normal output]
[more normal output]
[some error related to previous line]
[even more normal output]
```

Other wise you'd get:
```
stdout:
[normal output]
[more normal output]
[even more normal output]

stderr:
[some error related to some unknown line]
```
